From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robin Snyders <robin@snyders.xyz>
Date: Wed, 12 Aug 2026 00:05:00 +0200
Subject: [PATCH] arm64: dts: qcom: hotdog: load NXP NFC configuration

OxygenOS applies board-specific RF, TVDD and core settings before starting
discovery. Name the equivalent NCI configuration firmware.

Signed-off-by: Robin Snyders <robin@snyders.xyz>
---
 arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts b/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
--- a/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
@@ -543,8 +543,9 @@
 	status = "okay";
 
 	nfc@28 {
 		compatible = "nxp,nxp-nci-i2c";
 		reg = <0x28>;
+		firmware-name = "nxp/pn553-hotdog.nci";
 
 		interrupt-parent = <&tlmm>;
 		interrupts = <47 IRQ_TYPE_LEVEL_HIGH>;
-- 
2.50.1
